src/main/python/linuxband/gui/events/groove.py

```diff
@@ -112,7 +119,7 @@ def __update_groove_info(self, gr):
         """ Update description, author ... of the currently selected groove. """
         tb = self.__textbuffer2
         tb.set_text('')
-        start, end = tb.get_bounds() #@UnusedVariable
+        start, end = tb.get_bounds()
```

Comment:
Same as before.

Reply:
Also here, I would prefer to change that to `tb.get_bounds()[0]`.